### DISCUSSION TITLE
What does the solarfx2 label mean?

### DISCUSSION THREAD

**Discussion by AdamRJensen** [pvlib/pvlib-python#1373] [Q&A] [answered]
Most of the labels used are self-explanatory (e.g., bug, enhancement, and io). However, I'm having a hard time figuring out what the `solarfx2` label means. See example usage below or #589.

![image](https://user-images.githubusercontent.com/39184289/148621092-bb5a60d2-65cc-4b19-8a41-9b9e85571ebb.png)

**Comment by wholmgren** [comment, marked as answer]
It stands for Solar Forecasting 2, the Department of Energy program that funds Solar Forecast Arbiter (among other projects). It's a way for me to keep track of the impact of that funding.

**Reply by AdamRJensen** [reply]
I realized that there is a list of labels with descriptions [here](https://github.com/pvlib/pvlib-python/labels).

@wholmgren I suggest adding a description of solarfx2 to the list, similar to how the label SPI is described.
